app/decorators/petition_search.rb

```diff
@@ -0,0 +1,75 @@
+class PetitionSearch
+
+  delegate :previous_page, :next_page, :to => :petitions
+  delegate :current_page, :per_page, :to => :petition_results
+  delegate :total_entries, :total_pages, :to => :petition_results
+  delegate :each, :empty?, :offset, :size, :to => :petition_results
+  delegate :first, :length, :to_a, :to_ary, :to => :petition_results
+
+  def initialize(params)
+    @params = params
+  end
+
+  def state
+    State::SEARCHABLE_STATES.include?(@params[:state]) ? @params[:state] : 'open'
+  end
+
+  def search_term
+   @params[:q] || ''
```

Comment:
In the old implementation there was a bunch of search term sanitization that went on (start at ``Search#construct_search_string``).  Is it worth keeping that stuff?  At least for now as I assume that solr/sunspot won't do it for us automatically.  I know the implementation of any sanitisation will be different when we move to an alternate search backend, but we might still need to do something.